suite/tests/client-interface/drbbdup-drwrap-test.dll.c

```diff
@@ -125,7 +127,12 @@ event_module_load(void *drcontext, const module_data_t *mod, bool loaded)
         return;
     app_pc target = (app_pc)dr_get_proc_address(mod->handle, "wrapme");
     CHECK(target != NULL, "cannot find lib export");
-    bool res = drwrap_wrap(target, wrap_pre, wrap_post);
+    int flags = 0;
+#if 0 /* XXX i#5356: Enable once drbbdup handles the no-app-pc sentinel block. */
```

Comment:
IIRC `DRWRAP_REPLACE_RETADDR` is an important feature. Maybe change the XXX to a TODO?

Reply:
Done, but probably a moot point since days later there is a PR to solve it which might make it in before this one: PR #5381.